src/main/java/org/neo4j/gis/spatial/functions/SpatialFunctions.java

```diff
@@ -54,5 +62,38 @@ public Object asGeometry(@Name("geometry") Object geometry) {
 		return toNeo4jGeometry(null, geometry);
 	}
 
+	@UserFunction("spatial.convert.nativeToWkt")
```

Comment:
I wonder about the three part name. I assume you chose that to differentiate between the other methods that work with nodes or geometries. But perhaps we need to think of a more consistent naming scheme?

The word `native` seems to mean a Neo4j point, which in the other methods was called `neo4jGeometry`. Perhaps we should use the term `neo4j` everywhere, or `native` everywhere and not two different terms?